files/zh-cn/web/javascript/reference/global_objects/map/delete/index.md

```diff
@@ -23,18 +23,18 @@ myMap.delete(key);
 ### 返回值
 
 - Boolean
-  - : 如果 `Map` 对象中存在该元素，则移除它并返回* `true`*；否则如果该元素不存在则返回 `false`。
+  - : 如果 `Map` 对象中存在该元素，则移除它并返回 `true`；否则直接返回 `false`。
 
 ## 示例
 
-### 使用 `delete` 方法
+### 使用 `delete()` 方法
 
 ```js
-var myMap = new Map();
+const myMap = new Map();
 myMap.set("bar", "foo");
 
-myMap.delete("bar"); // 返回 true。成功地移除元素
-myMap.has("bar");    // 返回 false。"bar" 元素将不再存在于 Map 实例中
+console.log(myMap.delete("bar")); // 返回 true。成功地移除元素
+console.log(myMap.has("bar"));    // 返回 false。"bar" 元素将不再存在于 Map 实例中
```

Comment:
```suggestion
myMap.delete('bar'); // 返回 true。成功地移除元素
myMap.has('bar');    // 返回 false。"bar" 元素将不再存在于 Map 实例中
```

Reply:
Please check those unresolved suggestions, you can try to update the english documents, then please poke me for a review.

Or just apply those suggestions